core/trino-web-ui/src/main/resources/webapp/src/components/WorkerStatus.jsx

```diff
@@ -267,9 +267,9 @@ export class WorkerStatus extends React.Component {
             <div>
                 <table className="table">
                     <tbody>
-                        {Object.keys(queries).map((key) =>
-                            WorkerStatus.renderPoolQuery(key, queries[key][0], queries[key][1], size)
-                        )}
+                        {Object.keys(queries)
+                            .sort((a, b) => queries[b][0] - queries[a][0])
```

Comment:
(a, b) -> (queryA, queryB)

Reply:
Fixed to `([queryA, reservationsA], [queryB, reservationsB])` to use `Object.entries`.